disperser/dataapi/v2/accounts.go

```diff
@@ -102,3 +103,96 @@ func (s *ServerV2) FetchAccountBlobFeed(c *gin.Context) {
+//	@Failure	404			{object}	ErrorResponse	"error: Not found"
+//	@Failure	500			{object}	ErrorResponse	"error: Server error"
+//	@Router		/accounts/{account_id}/reservation/usage [get]
+func (s *ServerV2) FetchAccountReservationUsage(c *gin.Context) {
+	handlerStart := time.Now()
+	var err error
+
+	// Parse account ID
+	accountStr := c.Param("account_id")
+	if !gethcommon.IsHexAddress(accountStr) {
+		s.metrics.IncrementInvalidArgRequestNum("FetchAccountReservationUsage")
+		invalidParamsErrorResponse(c, errors.New("account id is not valid hex"))
+		return
+	}
+	accountId := gethcommon.HexToAddress(accountStr)
+	if accountId == (gethcommon.Address{}) {
+		s.metrics.IncrementInvalidArgRequestNum("FetchAccountReservationUsage")
+		invalidParamsErrorResponse(c, errors.New("zero account id is not valid"))
+		return
+	}
+
+	// Parse window parameter
+	window := 24 // default 24 hours
+	if windowStr := c.Query("window"); windowStr != "" {
+		parsedWindow, err := strconv.Atoi(windowStr)
+		if err != nil || parsedWindow <= 0 || parsedWindow > 72 {
+			s.metrics.IncrementInvalidArgRequestNum("FetchAccountReservationUsage")
+			invalidParamsErrorResponse(c, errors.New("window must be between 1 and 72 hours"))
+			return
+		}
+		window = parsedWindow
+	}
+
+	// Calculate reservation period
+	now := time.Now()
+	startTime := now.Add(-time.Duration(window) * time.Hour)
+
+	// Get period records for the specified window (limit 1000)
+	periodRecords, err := s.meteringStore.GetReservationBinUsage(c.Request.Context(), accountId, uint64(startTime.Unix()), 1000)
+	if err != nil {
+		s.metrics.IncrementFailedRequestNum("FetchAccountReservationUsage")
+		errorResponse(c, fmt.Errorf("failed to fetch period records for account (%s): %w", accountId.Hex(), err))
+		return
+	}
+
+	// Convert period records to response format
+	records := make([]PeriodRecord, len(periodRecords))
+	for i, record := range periodRecords {
+		if record == nil {
+			records[i] = PeriodRecord{
+				ReservationPeriod: 0,
+				UsageBytes:        0,
+			}
+		} else {
+			records[i] = PeriodRecord{
+				ReservationPeriod: record.Index,
+				UsageBytes:        record.Usage * 32,
+			}
+		}
+	}
+
+	response := &AccountReservationUsageResponse{
+		AccountId: accountId.Hex(),
+		Records:   records,
+	}
+
+	s.metrics.IncrementSuccessfulRequestNum("FetchAccountReservationUsage")
+	s.metrics.ObserveLatency("FetchAccountReservationUsage", time.Since(handlerStart))
+	c.Writer.Header().Set(cacheControlParam, fmt.Sprintf("max-age=%d", maxBlobFeedAge))
+	c.JSON(http.StatusOK, response)
+}
+
+// AccountReservationUsageResponse represents the response for account reservation usage
+type AccountReservationUsageResponse struct {
+	AccountId string         `json:"account_id"`
+	Records   []PeriodRecord `json:"records"`
+}
+
+// PeriodRecord represents a single period's usage record
+type PeriodRecord struct {
+	ReservationPeriod uint32 `json:"reservation_period"`
+	UsageBytes        uint64 `json:"usage_bytes"`
```

Comment:
usage is metered in unit of symbols (32bytes)

Reply:
removed symbols -> bytes conversion